src/components/overlay/pages/overlay-interactive.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DocsActiveBlock from 'components/DocsActiveBlock';
+import Overlay from '../Overlay';
+
+const Overlays = () => {
+  const settings = [];
+
+  return <div>
+    <DocsActiveBlock settings={settings}>
+      <div>
+        <Overlay/>
```

Comment:
we used it this way in the style guide, I even checked with much older versions (70-80)